testdata/invoices.go

```diff
@@ -0,0 +1,115 @@
+package testdata
+
+// GetInvoiceResponse example
+var GetInvoiceResponse = `{
```

Comment:
@ducnt114 can we please make this ones `const` and not vars, this to comply with #18 
```suggestion
const GetInvoiceResponse = `{
```